src/util/PrintUtil/PrintUtil.spec.ts

```diff
@@ -1,42 +1,43 @@
 /*eslint-env jest*/
 import OlImageWmsSource from 'ol/source/ImageWMS';
-import OlVectorSource from 'ol/source/Vector';
-import OlVectorLayer from 'ol/layer/Vector';
 import OlImageLayer from 'ol/layer/Image';
 
-import TestUtils from '../../../spec/TestUtils';
+// This causes issues via rbush depency
```

Comment:
Please note this as an issue.

Reply:
https://github.com/terrestris/react-geo-baseclient/issues/449